### ISSUE TITLE
TypeError: dump_all() got an unexpected keyword argument 'sort_keys'

### ISSUE THREAD

**Issue by IamSparky** [ultralytics/yolov5#414]
![image](https://user-images.githubusercontent.com/42636586/87542734-1a45ef00-c6c1-11ea-90ed-f34e928a1354.png)

**Got this error while running the train.py file in Google Colab**

the full link of the notebook is - https://colab.research.google.com/drive/10J3_S3_pjpvh55ZwoBnjK3xyX0zTRSSq?usp=sharing

## To Reproduce (REQUIRED)

Input:
```
!python train.py --img 1024 --batch 32 --epochs 10 --data /content/wheat.yaml --cfg models/yolov5s.yaml --name wm --nosave --cache
```

Output:
```
Apex recommended for faster mixed precision training: https://github.com/NVIDIA/apex
Namespace(batch_size=32, bucket='', cache_images=True, cfg='models/yolov5s.yaml', data='/content/wheat.yaml', device='', epochs=10, evolve=False, hyp='', img_size=[1024], multi_scale=False, name='wm', noautoanchor=False, nosave=True, notest=False, rect=False, resume=False, single_cls=False, weights='')
Using CUDA device0 _CudaDeviceProperties(name='Tesla K80', total_memory=11441MB)

2020-07-14 21:11:21.719560: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Start Tensorboard with "tensorboard --logdir=runs", view at http://localhost:6006/
Hyperparameters {'optimizer': 'SGD', 'lr0': 0.01, 'momentum': 0.937, 'weight_decay': 0.0005, 'giou': 0.05, 'cls': 0.58, 'cls_pw': 1.0, 'obj': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0.0, 'hsv_h': 0.014, 'hsv_s': 0.68, 'hsv_v': 0.36, 'degrees': 0.0, 'translate': 0.0, 'scale': 0.5, 'shear': 0.0}
Traceback (most recent call last):
  File "train.py", line 404, in <module>
    train(hyp)
  File "train.py", line 57, in train
    yaml.dump(hyp, f, sort_keys=False)
  File "/usr/local/lib/python3.6/dist-packages/yaml/__init__.py", line 200, in dump
    return dump_all([data], stream, Dumper=Dumper, **kwds)
TypeError: dump_all() got an unexpected keyword argument 'sort_keys'
```

## Expected behavior
It should start training the model

## Environment
I was running it on google colab Gpu

**Comment by glenn-jocher** [comment]
@soumochatterjee appears your pyyaml is not up to date. You should follow the **Setup** instructions in the official notebook. At the same time, removing the `sort_keys` arguments from the code does not appear to have any adverse effects.

**Comment by glenn-jocher** [comment]
<img width="993" alt="Screen Shot 2020-07-15 at 10 52 57 AM" src="https://user-images.githubusercontent.com/26833433/87578538-64ab7980-c689-11ea-8e9e-32040b3afc99.png">

**Comment by wwdok** [comment]
I met the exact same issue, i fixed it by upgrage PyYAML version to > 3.13, not include =3.13 :  `!pip install PyYAML==5.4.1`

**Comment by glenn-jocher** [comment]
@wwdok thanks for the comment! What environment did you encounter the error in? Perhaps we need to increase the pyYAML requirement.

**Comment by wwdok** [comment]
@glenn-jocher hi, i run it with latest colab just now, so the environment is the latest colab used.

**Comment by glenn-jocher** [comment]
@wwdok ok understood. It appears I made a poor design decision in reverting the yaml requirement. It works for me on my Colab, but if it doesn't work on your Colab then it may not work for others either and I should increase the version. I will update requirements.txt with this info.

**Comment by glenn-jocher** [comment]
@wwdok @seahawks8 PR #2030 is merged, which updates the pyYAML requirement to ==5.4.1. Please git pull to receive this update, and let us know if you run into any other issues.

Thank you for your feedback!

**Comment by bryanbocao** [comment]
> I met the exact same issue, i fixed it by upgrage PyYAML version to > 3.13, not include =3.13 : `!pip install PyYAML==5.4.1`

Thanks! It works!

**Comment by glenn-jocher** [comment]
@bryanbocao glad to hear that it's working for you now! If you have any more questions or run into any other issues, feel free to ask. Happy coding!
